Formula/macvim.rb

```diff
@@ -14,20 +14,29 @@ class Macvim < Formula
   end
 
   option "with-override-system-vim", "Override system vim"
+  option "with-gettext", "Build MacVim with +gettext."
```

Comment:
The `depends_on "gettext" => :optional` below will generate the option text, so we don't need this line.